.github/CODEOWNERS

```diff
@@ -13,11 +13,9 @@ tmp                                   @DFE-Digital/school-placements  @DFE-Digit
 vendor                                @DFE-Digital/school-placements  @DFE-Digital/track-pay
 
 app/*/claims                          @DFE-Digital/track-pay
-app/policies/claim_policy.rb          @DFE-Digital/track-pay
 spec/*/claims                         @DFE-Digital/track-pay
 config/locales/*/claims               @DFE-Digital/track-pay
 config/locales/*/claims.yml           @DFE-Digital/track-pay
-config/routes/claims.rb               @DFE-Digital/track-pay
```

Comment:
I'm assuming `config/routes/claims.rb` is covered by the existing `app/*/claims` rule, and that's why you've removed it. Is that right?

Reply:
Whoops. That was my bad. It won't be included. Adding back...

Reply:
Added back.